src/components/List/List.tsx

```diff
@@ -128,7 +128,10 @@ class List extends AutoControlledComponent<ReactProps<ListProps>, ListState> {
           const targetComponent = this.itemRefs[index] && this.itemRefs[index].current
           const targetDomNode = ReactDOM.findDOMNode(targetComponent) as any
 
-          targetDomNode && targetDomNode.focus()
+          if (targetDomNode) {
```

Comment:
Let's add a comment as to _why_ this special behavior is needed.  Accessibility requires arrow keys to focus the inner div of the LI, not the LI itself.  Correct?